targetconfig.json

```diff
@@ -32,6 +32,7 @@
         "Lessons": "lessons",
         "Courses": "courses",
         "Hardware": "hardware",
+        "How to Make a Game Videos": "videos",
```

Comment:
videos is too generic, that page should list all videos. I suggest having a specific page for the cat platformer.

Reply:
Yeah, that's what it was before, but I agree it's too generic

Reply:
Renamed that folder to "how-to-make-a-game/"